src/CTA.WebForms/FileConverters/CodeFileConverter.cs

```diff
@@ -41,8 +43,19 @@ public class CodeFileConverter : FileConverter
 
             try
             {
-                _fileModel = _webFormsProjectAnaylzer.AnalyzerResult.ProjectBuildResult?.SourceFileBuildResults?
-                    .Single(r => r.SourceFileFullPath.Equals(fullPath, StringComparison.OrdinalIgnoreCase))?.SemanticModel;
+                var sourcefileBuildResult = _webFormsProjectAnaylzer.AnalyzerResult.ProjectBuildResult?.SourceFileBuildResults?
+                    .Single(r => r.SourceFileFullPath.Equals(fullPath, StringComparison.OrdinalIgnoreCase));
+                var oldFileModel = sourcefileBuildResult.SemanticModel;
+                var oldTree = sourcefileBuildResult.SyntaxTree;
```

Comment:
Same as above, we will need a null conditional operator on `sourcefileBuildResult` here